modules/user-guide/pages/installing-vale-cli.adoc

```diff
@@ -19,13 +19,35 @@ To use the various Vale at Red Hat styles, use Vale version 2.30.0 or lower.
 
 * A recent version of the link:https://docs.asciidoctor.org/asciidoctor/latest/install/[Asciidoctor] Ruby gem is installed on your system.
 
+* Remove any previously installed `mczernek/vale` rpm package.
++
+[IMPORTANT]
+====
+The `mczernek/vale` copr package is no longer maintained and installing or updating Vale on Fedora version 39 with this method will result in a broken Vale installation.
+
+To remove the `mczernek/vale` copr installation, do the following:
+
+[source,terminal]
+----
+sudo dnf remove vale
+----
+====
+
 .Procedure
 
-. To install the Vale command-line tool on RHEL or Fedora, run the following command:
+. To install the Vale command-line tool on RHEL or Fedora, run the following commands:
 +
 [source,terminal]
 ----
-$ sudo dnf copr enable mczernek/vale && sudo dnf install vale
+wget https://github.com/errata-ai/vale/releases/download/v2.30.0/vale_2.30.0_Linux_64-bit.tar.gz
+
+tar -xvzf vale_2.30.0_Linux_64-bit.tar.gz vale
+
+mv vale ${HOME}/bin/vale
+
+rm vale_2.30.0_Linux_64-bit.tar.gz
+
+echo "Installed $(vale --version) to ${HOME}/bin/vale"
```

Comment:
I don't think this is necessary for commands that are manually run by the reader.

Reply:
`$(vale --version)` just to ensure that it got installed properly, and `${HOME}/bin/vale` in case they need to copy the location to an editor config :shrug:

Reply:
Fair enough. It's also easy to skip when running commands yourself. :)

Reply:
I like having this verification step. Nice touch, IMO.